docs/sources/release-notes/v2.5.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Grafana Mimir version 2.5 release notes"
+menuTitle: "V2.5 release notes"
+description: "Release notes for Grafana Mimir version 2.5"
+weight: 300
+---
+
+# Grafana Mimir version 2.5 release notes
+
+Grafana Labs is excited to announce version 2.5 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
```

Comment:
Like in previous release notes, could you spend few works about each item? See the [2.4 release notes](https://grafana.com/docs/mimir/latest/release-notes/v2.4/) as a reference for the format we adopted in previous releases.

Reply:
I have tried to write something up here, I also asked Jesse to take a look at it: https://github.com/grafana/mimir/pull/3580/files#diff-fddaf7aaf5533526b861513c83bb17e342adcfbec5e0c7c36bef8e0c47e49083R14-R29